pinot-connectors/pinot-spark-3-connector/README.md

```diff
@@ -0,0 +1,69 @@
+<!--
```

Comment:
Disclosure: I copied this documentation from the Spark2 based implementation since at this point the functionality is pretty much same. However it is likely they will diverge pretty soon so I think they deserve separate doc pages.